src/factory-plugin.ts

```diff
@@ -8,12 +8,19 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
+import * as theia from '@theia/plugin';
 import { FactoryTheiaClient } from "./factory-theia-client";
 import { CheWorkspaceProjectManager } from "./che-workspace-project-manager";
 
 export async function start() {
-    await (new FactoryTheiaClient()).onStart();
-    new CheWorkspaceProjectManager().onStart();
+    let projectsRoot = '/projects';
+    const projectsRootEnvVar = await theia.env.getEnvVariable('CHE_PROJECTS_ROOT');

```

Comment:
could this variable be provided by che-api( `@eclipse-che/plugin`  ?)
like `che.something.getProjectsRoot()` ?

Reply:
I think yes. But let it be as it is for now. It's another issue.